hr_timesheet_sheet_week_start_day/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=================================
+Hr Timesheet Sheet Week Start Day
```

Comment:
do you need me to change anything? Now the technical name is hr_timesheet_sheet_week_start_day, and the module name in openerp.py and README.rst is "Hr Timesheet Sheet Week Start Day".